charts/consul/templates/api-gateway-podsecuritypolicy.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if and .Values.apiGateway.enabled .Values.global.enablePodSecurityPolicies }}
```

Comment:
This is the PSP that will be applied to all `Gateway` deployments by default when `global.enablePodSecurityPolicies=true`. As I mentioned above, customers needing something more or less restrictive could create their own `PodSecurityPolicy` or modify this one and have the API gateway controller use it instead.

This is based on the PSP that we use for the API gateway controller with some slight modifications. Please review this for sensibility.